### ISSUE TITLE
[Linux] Mumble still auto-loads jackd even if you don't select JACK as the audio device

### ISSUE THREAD

**Issue by polyzium** [mumble-voip/mumble#3694]
Mumble still forces JACK to load.
```
polyzium@antergos-intel ~ % grep -rn .config/Mumble -e "PulseAudio"                                          
.config/Mumble/Mumble.conf:8:input=PulseAudio
.config/Mumble/Mumble.conf:11:output=PulseAudio
polyzium@antergos-intel ~ % mumble
Cannot connect to server socket err = No such file or directory
Cannot connect to server request channel
jackdmp 1.9.12
Copyright 2001-2005 Paul Davis and others.
Copyright 2004-2016 Grame.
Copyright 2016-2017 Filipe Coelho.
```
etc.

**Comment by davidebeatrici** [comment]
#3683

**Comment by davidebeatrici** [comment]
If you want to prevent Mumble from starting the JACK server, see #3674.

**Comment by polyzium** [comment]
Just what I need. Otherwise PulseAudio won't have access to the audio device because it's overtaken by JACK.
Better fix this ASAP because some of my buddies use Arch Linux where packages update almost instantly and they don't like fiddling around with configs.

Anyway, thank you. Closing.

**Comment by davidebeatrici** [comment]
You're welcome.

We will definitely fix the issue.

**Comment by davidebeatrici** [comment]
Fixed in #3826.
